pages/linux/upower.md

```diff
@@ -0,0 +1,24 @@
+# upower
+
+> System utility to provide battery information and statistics.
+> More information: <https://upower.freedesktop.org/>.
```

Comment:
```suggestion
> More information: <https://upower.freedesktop.org/docs/upower.1.html>.
```
I think we should directly point to the man page. Finding the home page from there is easier than the other way around.

Reply:
We only use `manned.org` if there isn't an official source.

Reply:
Actually, manned.org looks better, and since they have exactly the same content, I think we should use manned.org.  wdyt?

Reply:
The official page isn't just a man page, it is more

Reply:
Then let's use the official site 👍🏻